src/identity.js

```diff
@@ -417,7 +417,8 @@ export class Identity extends EventEmitter {
         const promiseFn = async (resolve, reject) => {
             const postProcess = (sessionData) => {
                 if (sessionData.error) {
-                    throw new SDKError('HasSession endpoint returned an error', sessionData.error);
+                    this.emit('error', sessionData.error);
+                    reject(new SDKError('HasSession failed', sessionData.error));
```

Comment:
The same idea should probably also be applied to https://github.com/schibsted/account-sdk-browser/blob/73a683546cb83e31c2c86a23eff96b80932cfa10/src/identity.js#L453-L456

Reply:
True, and thanks for explaining why this happens :)

Reply:
Fixed.